lib/kinetic_cafe/error_module.rb

```diff
@@ -79,6 +84,13 @@ def initialize(*args)
       query = options.delete(:query)
       @i18n_params.merge!(query: stringify(query)) if query
       @i18n_params.merge!(options)
+
+      level = options.delete(:level) || default_level
+      if level.kind_of?(Fixnum)
+        @level = level
+      else
+        @level = ::Logger::Severity.const_get(level.upcase)
```

Comment:
What if `Logger` isn’t loaded? I think that avoiding the `const_get` is a better option here; instead recommend symbolic names.